apps/bot/db/actions/users.ts

```diff
@@ -143,3 +143,17 @@ export const removePointsFromUser = async (
   type: keyof typeof POINTS_REWARDS,
   trx: TransactionDB | KyselyDB = db,
 ) => updatePointsBySum(userId, -POINTS_REWARDS[type], trx)
+
+export const getCorrectAnswersCount = (userId: string) => {
+  return db
+    .selectFrom('users')
+    .where('snowflakeId', '=', userId)
+    .select(['answersCount'])
+    .executeTakeFirst()
+}
+
+export const isUserProfilePublic = async (user: GuildMember) => {
+  if (env.PUBLIC_PROFILE_ROLE_ID)
+    return user.roles.cache.has(env.PUBLIC_PROFILE_ROLE_ID)
+  else return true // If env doesn't exist, consider it as private
```

Comment:
This comment doesn't seem to fit the name of the function, does it? If you consider the user to be private, it should return `false`, while it should return `true` if you consider the user to be public.